tools/doc/html.js

```diff
@@ -162,6 +168,40 @@ function analyticsScript(analytics) {
   `;
 }
 
+function altDocs(filename) {
+  let html = '';
+
+  if (!docCreated) {
+    console.error('Failed to add alternative version links');
+    return html;
+  }
+
+  function lte(v) {
+    if (docCreated[1] > v[0])
+      return false;
+    if (docCreated[1] < v[0])
+      return true;
+    return docCreated[2] <= v.substr(2, 2);
+  }
+
+  const host = 'https://nodejs.org';
+  const href = (v) => `${host}/docs/latest-v${v}/api/${filename}.html`;
+  const a = (v) => `<a href="${href(v)}">v${v}</a>`;
+  const as = (vs) => vs.filter(lte).map(a).join(' / ');
+
+  html += 'View another version of this page <b>Current:</b> ' + a('7.x');
```

Comment:
so one concern I have here is what will this look like when Node 8 is released? It will be released as a "Current" release line for a while before going LTS. With that, there will be a few months of layover where Node 7 is still also a "Current" release.